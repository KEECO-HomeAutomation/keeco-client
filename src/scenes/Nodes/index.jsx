import React from 'react';

const Nodes = () => {
	return <p>Nodes</p>;
};

export default Nodes;
