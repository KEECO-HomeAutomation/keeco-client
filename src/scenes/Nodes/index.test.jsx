import React from 'react';
import { shallow } from 'enzyme';

import Nodes from './index';

describe('<Nodes />', () => {
	test('Should render without errors', () => {
		const comp = shallow(<Nodes />);
		expect(comp.length).toBe(1);
	});

	test('Should match snapshot', () => {
		expect(shallow(<Nodes />)).toMatchSnapshot();
	});
});
