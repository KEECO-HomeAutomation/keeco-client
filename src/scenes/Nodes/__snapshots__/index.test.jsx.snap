// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nodes /> Should match snapshot 1`] = `
<p>
  Nodes
</p>
`;
